src/server/src/routes/workExperience/index.ts

```diff
@@ -80,40 +73,19 @@ router.post(
 /******************************************************************************
  *            GET Request - Read - /api/workExperience/read
  ******************************************************************************/
-
-interface WorkExperienceReadRequest extends Request {
-    query: {
-        title: string;
-        startDate: string;
-        endDate: string;
-        employer: string; 
-        description: string; 
-        numOfItems: string;
-    };
-}
-
 router.get(
     '/read',
     passport.authenticate('jwt', { session: false }),
-    validationMiddleware({ querySchema: workExperienceReadSchema }),
-    async (req: WorkExperienceReadRequest, res: Response) => {
-        const { title, numOfItems } = req.query;
-        let { startDate, endDate, employer, description  } = req.query;
-
+    async (req: Request, res: Response) => {
+        const { studentId } = req.params; 
         try {
-            if (!startDate) {
-                startDate = '01/01/3000';
-            }
-
-            const [workExperiences, workExperiencesCount] = await getWorkExperiences(
-                title,
-                employer,
-                description, 
-                startDate,
-                endDate, 
-                parseInt(numOfItems)
-            );
-            return res.status(OK).json({ workExperiences, workExperiencesCount }).end();
+            const workExperiences = await getWorkExperiences(studentId);
```

Comment:
you can convert the studentId to a number doing `parseInt(studentId, 10)` The 10 is passed there to make the conversion in base 10 numbers btw.